packages/augur-core/source/contracts/reporting/Universe.sol

```diff
@@ -250,7 +250,8 @@ contract Universe is IUniverse {
         uint256 _windowId = getDisputeWindowId(_timestamp, _initial);
         if (disputeWindows[_windowId] == IDisputeWindow(0)) {
             uint256 _duration = getDisputeRoundDurationInSeconds(_initial);
-            uint256 _startTime = _timestamp.div(_duration).mul(_duration).add(Reporting.getDisputeWindowBufferSeconds());
+            uint256 _buffer = Reporting.getDisputeWindowBufferSeconds();
```

Comment:
I missed the reason for `_buffer`. what is that?

Reply:
So that markets ending at midnight (very typical) have a one hour window where if the creator reports  quickly the dispute window will start in less than an hour.